tests/events/32-stall-despite-clock-trig.t

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2017 NIWA
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test stall does not wait for unrelated clock trigger.
+. "$(dirname "$0")/test_header"
+set_test_number 3
+
+install_suite "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
+run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}"
+run_fail "${TEST_NAME_BASE}-run" timeout 60 cylc run --debug "${SUITE_NAME}"
+sed -n 's/^.* WARNING - //p' "${SUITE_RUN_DIR}/log/suite/log" \
+    >"${SUITE_RUN_DIR}/log/suite/log.edited"
+TODAY="$(date '+%Y%m%d')"
```

Comment:
Sorry. Now fixed. (It is good to collaborate with colleagues living in the opposite side of the world! This sort of textbook mistakes will be a lot harder to spot otherwise.)